### DISCUSSION TITLE
Welcome to Rawon Discussion

### DISCUSSION THREAD

**Discussion by mzrtamp** [stegripe/rawon#170] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
Do slash commands! That will be amazing because discord release it and yea 

**ngl really cool**

**Reply by mzrtamp** [reply]
#451

**Comment by StefanPrisacariu** [comment]
How do i prevent the bot from leaving immediately after the queue is empty ?

**Reply by Mednoob** [reply]
@StefanPrisacariu You can set the `STAY_IN_VC_AFTER_FINISHED` value in .env file to `yes` to make the bot stay after queue finished playing

**Comment by StefanPrisacariu** [comment]
One more question, i just saw that i haven't updated the script, i updated it and for some reason it doesn't work, in the .env file do i have to add quotation marks for every setting ?
![image](https://user-images.githubusercontent.com/94453323/148382513-c74ecedf-f55b-40b7-8e7b-55a19a727e1d.png)

it just stays like that, i have followed your guide to host it via glitch and google script again and it still doesn't work

UPDATE :

![image](https://user-images.githubusercontent.com/94453323/148382653-e2dd4c8a-ee62-4837-94a4-b5207755c06f.png)

UPDATE 2 : 

Got some errors 

![image](https://user-images.githubusercontent.com/94453323/148389110-5c59035c-9f2c-47b6-9e3e-28ef76002729.png)

**Reply by shylockza** [reply]
Hosting on Glitch is broken for more than a year...

**Comment by 21Z** [comment]
Is replit supported now? (v3)

**Reply by Mednoob** [reply]
Oh, no. I forgot about replit. I'm sorry. I'll try to check it later when I have free time.

**Comment by RoyaleOne** [comment]
What eval commands exist? And how can i add my own if possible (to view ram usage and cpu usage)

**Reply by fab1ofx** [reply]
For the developer category there is an eval command, for the variable: `DEVS=<your discord id>` in .env
